README.md

```diff
@@ -995,12 +1005,12 @@ const [epsilon, setEpsilon] = React.useState(0.001);
 
 The argument of the `useState` function is the initial value. The `epsilon` variable contains the current value for epsilon and `setEpsilon` is a function to set epsilon to a new value.
 
-The input tag in the form will call the `onChange` function with a event object. We need to extract the user input from the event and pass it to `setEpsilon`.
+The input tag in the form will call the `onChange` function with a event object. We need to extract the user input from the event and pass it to `setEpsilon`. The value should be a number, so we use `*1` to cast the string from the event to a number.
```

Comment:
Can you use Number function for casting?
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number

Reply:
Certainly